testsuite/features/step_definitions/setup_steps.rb

```diff
@@ -488,16 +488,31 @@
       log "The MU repository #{unique_repo_name} was already created, we will reuse it."
     else
       content_type = deb_host?(client) ? 'deb' : 'yum'
-      steps %(
-        When I follow the left menu "Software > Manage > Repositories"
-        And I follow "Create Repository"
-        And I enter "#{unique_repo_name}" as "label"
-        And I enter "#{repo_url.strip}" as "url"
-        And I select "#{content_type}" from "contenttype"
-        And I click on "Create Repository"
-        Then I should see a "Repository created successfully" text or "The repository label '#{unique_repo_name}' is already in use" text
-        And I should see "metadataSigned" as checked
-      )
+
+      if %w[slmicro60_minion slmicro61_minion].include?(client)
```

Comment:
I don't think we should rely on hostnames, I think we should decouple the network infrastructure from our test framework decisions.

Reply:
These are not hostnames, but symbolic names - fully under our control and independant from infra.

But ok to remain consistent with what we already do.